homeassistant/components/mobile_app/notify.py

```diff
@@ -134,10 +134,6 @@ def targets(self):
                     response = await self._session.post(push_url, json=data)
                     result = await response.json()
 
-                if response.status == 201:
-                    log_rate_limits(self.hass, entry_data[ATTR_DEVICE_NAME], result)
-                    return
```

Comment:
This won't work. Now you're going to process it as an error! And you don't log the rate limits anymore. 

Did you test this?

You should do `continue` instead of `return`.

Reply:
Yes, I tested it, and it works in that it didn't abort the loop early and miss successive devices, but you're correct that it treats it as an error, doesn't log rate limits, and is generally sloppy.  It looks like HTTP status codes 200/201/202 could all be valid responses to the request, so I'm inclined to also treat 200 and 202 as valid options.  I'll submit an update.